device-types/APC/SMX1500RM2UNC.yaml

```diff
@@ -0,0 +1,37 @@
+---
+manufacturer: APC
+model: SMX1500RM2UNC
+slug: apc-smx1500rm2unc
+part_number: SMX1500RM2UNC
+comments: |
+  APC Smart-UPS X, Line Interactive, 1500VA, Rack/Tower LCD 120V with Network Card
+u_height: 2
```

Comment:
Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
If you have any questions, please let us know :)